darts/tests/models/forecasting/test_regression_models.py

```diff
@@ -1289,6 +1289,49 @@ def test_historical_forecast(self, mode):
         )
         assert len(result) == 21
 
+    @pytest.mark.parametrize("mode", [True, False])
```

Comment:
The added logic is the same for both modes, so I think we can simply check only one of the two